app/views/root/gem_layout_full_width_old_header.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= render partial: 'gem_base', locals: {
+  full_width: true,
+  show_explore_header: false,
```

Comment:
Just so we're aware - setting any parameter to `false` that uses `||=` to assign in `gem_base` template will get overwritten by the default.

So when `show_explore_header` gets set to true by default in the `gem_base` template, the template needs to be updated so it doesn't overwrite this parameter when `false` is passed in.

Reply:
👍